src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -24,24 +26,23 @@ const fakeIssues: Issue[] = [
 describe('generateConfluenceMarkdown function', () => {
     it('should pass in a dependency with no issues', () => {
         const mapWithoutIssues: Map<string, Issue[]> = new Map();
-        // const noIssues: Issue[] = [];
+        const noIssues: Issue[] = [];
         const dependency = 'TestDependency';
-        const oneDependency = mapWithoutIssues.set(dependency, fakeIssues);
-
+        const oneDependency = mapWithoutIssues.set(dependency, noIssues);
         const results = generateConfluenceMarkdown(oneDependency, 30);
-        expect(results).toContain([]); // expecting it to fail because fakeIssues has two issues...
+
+        expect(results).toContain(noIssues);
```

Comment:
I don't think toContain([]) is meaningful. We could probably say toNotContain(...) and pass in something that would normally only appear if there was at least one issue. And/or we could split the results by a newline, and expect there to be a specific number of lines of output.